src/themes/clean/templates/core/news/index.html

```diff
@@ -13,7 +13,7 @@ <h1>{{ journal_settings.news.news_title }}</h1>
             <div class="col-md-12">
                 <h4>{{ item.title }}</h4>
                 <h6>{{ item.byline }} on {{ item.posted|date:"Y-m-d" }}</h6>
-                <p>{{ item.body|striptags|truncatesmart:400 }}</p>
+                <p>{{ item.body|striptags|truncatewords_html:50 }}</p>
```

Comment:
striptags should be dropped and safe put in its place as with others.

Reply:
good catch.  done & tested.

Reply:
Nice!